test/averageRating.spec.js

```diff
@@ -0,0 +1,188 @@
+import { describe, expect, test } from "@jest/globals";
+import { getAverageRating } from "../src/api";
+
+const mockId = 2;
+const mockImdb = "tt2953050";
+const mockImdbRating = 7.8;
+
+//Function to calculate average rating
+function mockMath(mockRating, mockImdbRating) {
+  if (mockRating.data.length >= 5) {
+    let sumOfAll = 0;
+    const mockData = mockRating.data;
+
+    mockData.forEach((int) => {
+      sumOfAll += int.attributes.rating;
+    });
+    const mockAnswer = sumOfAll / mockRating.data.length;
+    return mockAnswer;
+  } else {
+    return mockImdbRating;
+  }
+}
```

Comment:
This function just duplicates the logic you're trying to test. In that sense, you are not really testing the logic, but you're testing that your program code is identical to this code.